testapps/php56_extensions/composer.json

```diff
@@ -1,6 +1,33 @@
 {
     "require": {
         "php": "^5.6",
-        "phpunit/phpunit": "4.8.*"
+        "phpunit/phpunit": "4.8.*",
```

Comment:
It's not entirely accurate that we always have the latest versions of extensions. I think it would make sense to bail out early and tell them to specify "*" as the version because we don't support arbitrary versions.